src/components/sbb-autocomplete/readme.md

```diff
@@ -0,0 +1,106 @@
+The `sbb-autocomplete` is a component that can be bonded to any text input (`trigger`) to display a panel of suggested options.  
+The panel appears below/above the `origin` element. 
+
+## Usage
+
+If not explicitly set, the `sbb-autocomplete` will look for an `sbb-form-field` ancestor as `origin`.
+
+Also, if not explicitly set, it will look for an `input` in `origin` to use as `trigger`.
+
+```html
+<!-- Origin element -->
+<sbb-form-field label="Label">
+
+  <!-- Trigger element -->
+  <input placeholder="Trigger element" />
+
+  <sbb-autocomplete>
+    <sbb-option icon-name="clock-small" value="Option 1"> Option 1 </sbb-option>
+    <sbb-option icon-name="clock-small" value="Option 2"> Option 2 </sbb-option>
+    <sbb-divider />
+    <sbb-option icon-name="clock-small" value="Option 3"> Option 3 </sbb-option>
+  </sbb-autocomplete>
+</sbb-form-field>
+```
+Or, you can specify a different `origin` and `trigger`
+
+```html
+<!-- Origin element -->
+<div id="autocomplete-origin">Another origin</div>
+
+<!-- Trigger element -->
+<input id="autocomplete-txt" placeholder="Another trigger element" />
+
+<sbb-autocomplete origin="#autocomplete-origin" trigger="#autocomplete-txt">
```

Comment:
I might be wrong here, feel free to correct me and share your thoughts.

When I write components for a broad number of users, I always expect them to have situations that we'd never think of. With this in mind, I would always go for the freest solution.

In this specific case, I intended the 'origin' and 'trigger' as selectors rather than ids.
In fact,  I'm using them directly in querySelector(...)

However, **consistency** with other components is much more important than my opinion.
If in other implementations, we expect similar properties to be ids, I'll adapt and change it

Reply:
Yes, elsewhere we we expect similar properties to be ids. In addition to that, I see from the specs that the origin can be a string or even an `HTMLElement`, might be the same for the trigger (?)

> Provide a possibility `@Prop() origin: string | HTMLElement` to define another origin (where the overlay is attached)

Therefore I'd suggest to take, for example, the `sbb-menu` as reference where we also check whether the trigger is a string or an HTMLElement in the `configure()` method.

Reply:
Got it. 
Atm, both origin and trigger can be string or HTMLElement ( already took inspiration from the menu :D )